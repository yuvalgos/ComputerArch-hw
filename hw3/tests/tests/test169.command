./dflow_calc tests/test169.dat tests/test169.in d144 p84 d192 p85 p145 d34 p196 p141 d42 p101 p120 p55 p139 p86 d15 d147 d195 p102 d9 p0 p151 p88 p73 d38 p82 p62 p125 d112 d190 d42 d3 p149 d95 d150 d26 d186 p37 p64 d37 p66 d105 p2 p64 d24 d156 d126 d56 p172 d57 d58 p148 p153 d42 d176 d82 d53 d137 d129 p82 p16 p20 p96 d150 p172 d0 p198 d76 p62 d106 p115 d105 p166 p162 d177 d53 d23 p153 d108 d84 p81 p136 d190 p181 d165 d115 d126 p53 p191 p195 p196 p125 d119