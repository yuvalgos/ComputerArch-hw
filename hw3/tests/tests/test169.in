7 10 28 0
8 12 4 17
0 6 7 9
0 20 25 6
0 14 14 31
3 23 31 23
7 14 28 2
1 14 24 1
4 24 1 15
3 29 1 28
6 30 6 20
8 26 30 31
4 17 9 23
2 1 13 24
1 28 14 14
8 9 4 21
5 17 6 31
2 7 23 0
0 22 16 28
5 11 2 25
5 17 6 6
0 20 18 23
0 2 20 1
7 10 25 6
3 15 0 10
4 11 21 21
8 13 3 25
0 25 13 2
6 10 10 3
5 6 14 9
4 1 11 20
5 13 22 17
2 1 22 4
8 24 12 17
4 27 21 8
0 4 11 13
7 29 30 3
7 17 30 11
4 2 28 11
3 12 12 14
3 21 16 30
8 23 17 15
6 12 15 15
4 12 2 0
6 29 3 28
0 21 0 25
2 31 7 24
4 14 11 14
1 25 10 30
3 7 28 27
1 2 11 15
3 21 12 7
2 20 26 17
0 27 7 25
7 21 27 23
5 17 4 5
1 9 31 14
4 7 12 17
3 0 3 14
8 20 3 1
2 15 11 26
0 22 16 2
7 1 11 19
2 5 23 15
4 15 21 3
6 11 17 27
5 3 9 26
1 3 20 13
2 27 16 26
5 3 5 16
2 20 30 11
6 20 2 26
2 13 29 5
1 25 13 20
0 23 2 23
8 13 1 30
4 23 30 15
1 29 4 26
4 14 19 14
8 17 12 7
7 19 30 15
8 8 18 12
2 6 10 2
8 18 24 29
0 10 30 7
2 5 13 22
7 12 24 24
2 19 3 6
6 26 28 8
5 30 0 23
5 13 17 14
1 23 1 17
7 28 22 10
5 23 12 24
3 30 27 3
3 13 3 15
3 19 25 15
4 20 25 6
0 12 3 15
8 13 18 28
3 29 13 14
2 4 28 27
7 6 24 17
3 5 22 27
2 15 5 22
6 28 14 4
4 10 25 5
5 9 0 20
4 18 13 26
3 23 25 10
3 0 25 5
2 1 16 9
3 14 17 24
4 0 25 24
1 21 27 29
2 27 30 21
0 10 9 25
4 23 24 29
5 26 17 13
4 12 17 14
5 1 14 4
2 6 27 27
6 15 12 8
5 8 14 28
5 26 12 15
1 24 6 26
0 15 29 15
8 2 13 29
4 27 14 22
8 24 23 17
0 7 15 5
2 7 17 27
1 14 27 7
8 26 10 19
2 30 17 27
4 13 13 28
4 5 6 18
0 12 21 0
8 9 22 16
0 27 0 1
6 21 26 6
6 29 26 6
2 5 7 19
2 12 18 16
4 29 13 15
6 25 27 5
2 1 24 23
5 11 15 13
3 23 16 9
8 11 9 26
4 29 30 2
4 23 7 19
4 25 5 15
4 17 21 1
8 27 12 27
7 23 27 15
7 5 24 22
1 2 14 9
7 8 10 28
3 9 21 8
3 22 18 24
5 2 29 24
4 3 6 0
0 20 1 30
2 8 25 13
0 21 16 23
0 19 25 30
1 19 7 23
3 2 22 22
5 21 17 12
3 15 28 19
7 22 14 9
1 19 19 13
8 28 28 6
5 1 17 0
0 18 28 16
4 19 1 26
3 19 9 0
7 15 14 9
4 9 13 25
6 22 14 30
7 4 6 26
8 18 12 3
1 22 29 25
2 30 14 24
6 16 16 24
5 16 17 3
5 29 12 31
3 12 31 8
8 12 28 8
4 18 25 6
1 22 6 2
5 26 0 1
0 15 9 21
1 9 6 18
8 25 12 4
1 28 27 29
8 22 5 5
0 16 5 5
5 4 21 25
