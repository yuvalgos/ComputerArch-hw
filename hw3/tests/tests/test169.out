Reading the opcodes latency info from tests/test169.dat ... Got latency for 9 opcodes
Reading the program file tests/test169.in ... Found 200 instructions
getProgDepth()==200
getInstDeps(144)=={135,126}
getDepDepth(84)==54
getInstDeps(192)=={113,174}
getDepDepth(85)==53
getDepDepth(145)==105
getInstDeps(34)=={-1,-1}
getDepDepth(196)==137
getDepDepth(141)==125
getInstDeps(42)=={24,24}
getDepDepth(101)==75
getDepDepth(120)==84
getDepDepth(55)==30
getDepDepth(139)==97
getDepDepth(86)==46
getInstDeps(15)=={-1,-1}
getInstDeps(147)=={126,135}
getInstDeps(195)=={189,181}
getDepDepth(102)==77
getInstDeps(9)=={-1,-1}
getDepDepth(0)==0
getDepDepth(151)==100
getDepDepth(88)==56
getDepDepth(73)==73
getInstDeps(38)=={14,25}
getDepDepth(82)==38
getDepDepth(62)==11
getDepDepth(125)==125
getInstDeps(112)=={79,33}
getInstDeps(190)=={152,121}
getInstDeps(42)=={24,24}
getInstDeps(3)=={-1,2}
getDepDepth(149)==125
getInstDeps(95)=={69,64}
getInstDeps(150)=={134,127}
getInstDeps(26)=={-1,-1}
getInstDeps(186)=={153,162}
getDepDepth(37)==21
getDepDepth(64)==82
getInstDeps(37)=={10,25}
getDepDepth(66)==54
getInstDeps(105)=={78,101}
getDepDepth(2)==0
getDepDepth(64)==82
getInstDeps(24)=={-1,23}
getInstDeps(156)=={129,61}
getInstDeps(126)=={100,122}
getInstDeps(56)=={46,47}
getDepDepth(172)==159
getInstDeps(57)=={43,55}
getInstDeps(58)=={-1,47}
getDepDepth(148)==56
getDepDepth(153)==144
getInstDeps(42)=={24,24}
getInstDeps(176)=={174,133}
getInstDeps(82)=={28,50}
getInstDeps(53)=={49,48}
getInstDeps(137)=={114,113}
getInstDeps(129)=={117,79}
getDepDepth(82)==38
getDepDepth(16)==2
getDepDepth(20)==2
getDepDepth(96)==93
getInstDeps(150)=={134,127}
getDepDepth(172)==159
getInstDeps(0)=={-1,-1}
getDepDepth(198)==158
getInstDeps(76)=={10,64}
getDepDepth(62)==11
getInstDeps(106)=={73,103}
getDepDepth(115)==108
getInstDeps(105)=={78,101}
getDepDepth(166)==125
getDepDepth(162)==118
getInstDeps(177)=={159,113}
getInstDeps(53)=={49,48}
getInstDeps(23)=={-1,2}
getDepDepth(153)==144
getInstDeps(108)=={99,88}
getInstDeps(84)=={10,57}
getDepDepth(81)==33
getDepDepth(136)==118
getInstDeps(190)=={152,121}
getDepDepth(181)==125
getInstDeps(165)=={-1,155}
getInstDeps(115)=={94,114}
getInstDeps(126)=={100,122}
getDepDepth(53)==41
getDepDepth(191)==153
getDepDepth(195)==164
getDepDepth(196)==137
getDepDepth(125)==125
getInstDeps(119)=={79,112}
